modules/web/js/ballerina/views/connector-declaration-view.js

```diff
@@ -184,5 +184,24 @@ define(['lodash','d3', 'jquery', './ballerina-view', './../ast/connector-declara
             d3Element.attr("stroke-width", 0);
         };
 
+        ConnectorDeclarationView.prototype.createPropertyPane = function() {
```

Comment:
We have removed the `connector-declaration-view` module since we moved to React and replaced it with `connector-declaration.jsx` module. I will look into adding necessary doc comments to the instance methods there.